lib/python/rose/config_editor/keywidget.py

```diff
@@ -37,25 +37,18 @@ class KeyWidget(gtk.VBox):
     """This class generates a label or entry box for a variable name."""
 
     FLAG_ICON_MAP = {
-             rose.config_editor.FLAG_TYPE_DEFAULT:
-                  gtk.STOCK_INFO,
-             rose.config_editor.FLAG_TYPE_ERROR:
-                  gtk.STOCK_DIALOG_WARNING,
-             rose.config_editor.FLAG_TYPE_FIXED:
-                  gtk.STOCK_DIALOG_AUTHENTICATION,
-             rose.config_editor.FLAG_TYPE_OPT_CONF:
-                  gtk.STOCK_INDEX,
-             rose.config_editor.FLAG_TYPE_OPTIONAL:
-                  gtk.STOCK_ABOUT,
-             rose.config_editor.FLAG_TYPE_NO_META:
-                  gtk.STOCK_DIALOG_QUESTION}
+        rose.config_editor.FLAG_TYPE_DEFAULT: gtk.STOCK_INFO,
+        rose.config_editor.FLAG_TYPE_ERROR: gtk.STOCK_DIALOG_WARNING,
+        rose.config_editor.FLAG_TYPE_FIXED: gtk.STOCK_DIALOG_AUTHENTICATION,
+        rose.config_editor.FLAG_TYPE_OPT_CONF: gtk.STOCK_INDEX,
+        rose.config_editor.FLAG_TYPE_OPTIONAL: gtk.STOCK_ABOUT,
+        rose.config_editor.FLAG_TYPE_NO_META: gtk.STOCK_DIALOG_QUESTION}
```

Comment:
As with Ben's above comment, this'd be more readable and make more sense if the final brace was on a separate line.